ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -5034,6 +5061,16 @@ static int greeting_handler(ldmsd_req_ctxt_t reqc)
 	return 0;
 }
 
+static int bad_req_handler(ldmsd_req_ctxt_t reqc)
+{
+	size_t cnt;
+	reqc->errcode = EBADRQC;
+	cnt = Snprintf(&reqc->line_buf, &reqc->line_len,
+			"LDMSD received bad request.");
```

Comment:
Maybe "malformed" instead of bad? It maybe gives more of a hint about what was wrong with the request.

Reply:
I think this is the only change left to address. I'm inclined to keep "bad", as the error returned is literally EBADRQC, and malformed could limit the function's perceived use in the future. However, if the general thought is that this error reporting function will only ever be used for this particular use case, then I don't have any qualms changing it to malformed.